.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: minimal
+language: generic
```

Comment:
Historically `minimal` was required to get more disk space on the builders to avoid blowing the disk space limits on the Android builder, but it looks like `generic`  only has 1GB less of space (24 vs 25 GB) and I think we've mitigated this through other means. Getting out that ruby business sounds good!